evaluate_model.py

```diff
@@ -0,0 +1,57 @@
+
+from pipeline import load_pipeline, load_recordings, get_epochs
+from sklearn.metrics import confusion_matrix
+import numpy as np
+
+
+def main():
+    model_subject_name = 'Haggai2'
+    data_subject_name = 'Haggai2'
+
+    # model
+    pipeline = load_pipeline(model_subject_name)
+
+    # data
+    raw, params = load_recordings(data_subject_name)
+    epochs, labels = get_epochs(raw, params["trial_duration"])
+    epochs = epochs.get_data()
+
+    # evaluate
+    predictions = pipeline.predict(epochs)
+
+    # statistics
+    conf_matrix = confusion_matrix(labels, predictions)
+    print('confusion matrix (row=label, column=prediction):')
+    print(conf_matrix)
+
+    rates = calculate_true_and_false_rates(conf_matrix)
+    print('true positive and false positive rates (row=label, column=true or false):')
+    print(rates)
+
+
+def calculate_true_and_false_rates(conf_matrix):
```

Comment:
Nice function :) I think
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html

does something similar if you want to compare